guide/src/format/configuration/renderers.md

```diff
@@ -159,6 +159,7 @@ The following configuration options are available:
   navigation links and script/css imports in the 404 file work correctly, even when accessing
   urls in subdirectories. Defaults to `/`. If `site-url` is set,
   make sure to use document relative links for your assets, meaning they should not start with `/`.
+- **use-site-url-as-root:** Prepend the `site_url` in links with absolute path.
```

Comment:
Yes, done,

Reply:
> links with absolute path

As a user, I would wonder "what links are those"? It could be

* links generated by templates, such as those referencing style sheets and JavaScript code
* links generated between pages of the book
* links _I insert_ using Markdown syntax